app/src/main/java/com/simple/checkstyledemo/MainActivity.java

```diff
@@ -1,13 +1,23 @@
 package com.simple.checkstyledemo;
 
-import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.app.Application;
```

Comment:
**Reporter**: PMD
**Rule**: UnusedImports
**Severity**: WARN
**File**: app/src/main/java/com/simple/checkstyledemo/MainActivity.java L5

Import Statements https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/imports.html#UnusedImports

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*